docs/api/python/python.rst

```diff
@@ -28,11 +28,33 @@ If you would like a more feature-complete---yet verbose---python API, with the a
 You can compare examples using the two APIs by visiting the :doc:`examples page <../../examples/quickstart>`.
 
 
-Installation
+Installation (Except MacOS with Apple Silicon)
```

Comment:
```suggestion
Installation (x86-64 variants of Linux and macOS)
```
The instructions don't work for Windows or ARM versions of Linux, so it is better to explicitly list what's supported.